packages/VendorLocker/NodeVendorLocker/ClassMethodReturnTypeOverrideGuard.php

```diff
@@ -48,12 +48,14 @@ public function __construct(
 
     public function shouldSkipClassMethod(ClassMethod $classMethod, Scope $scope): bool
     {
-        // 1. skip magic methods
```

Comment:
dropped the comments as they did not provide any information, the method call wouldn't already convey on its own